tests/test_interface_unversioned.py

```diff
@@ -653,3 +655,66 @@ def test_move(tmpdir, src_path, dst_path, interface):
     # clean up
 
     interface.remove_file(dst_path)
+
+
+def test_validate(tmpdir):
+
+    class BadChecksumBackend(audbackend.backend.FileSystem):
+        r"""Return random checksum."""
+        def _checksum(
+                self,
+                path: str,
+        ) -> str:
+            return ''.join(random.choices(
+                string.ascii_uppercase + string.digits,
+                k=33,
+            ))
+
+    path = audeer.touch(tmpdir, '~.txt')
+    error_msg = 'Execution is interrupted because'
+
+    interface = audbackend.interface.Unversioned(
```

Comment:
To be on the safe side here, we should maybe add the interface as parameter of the test and than test for all available interfaces?

Reply:
Currently, we have two interfaces `Unversioned` and `Versioned` which we test in separate test files. The test you see here is repeated for `Versioned` in `test_interface_versioned.py`. Since the methods expect different arguments, we cannot easily make the interface class a parameter.

Reply:
Ok, then let's stick with the current solution.
Copy and pasting the code to the other test files (at some point we will also have an interface for the legacy functions),
seems also not so nice.

Reply:
> at some point we will also have an interface for the legacy functions

Since the legacy methods have the same arguments as the `Versioned` interface, it should be possible to test them mutually.